umake/ui/cli/__init__.py

```diff
@@ -157,6 +157,60 @@ def mangle_args_for_default_framework(args):
     return result_args
 
 
+def list_frameworks(listing, detailled=True):
```

Comment:
I begin to take in consideration your suggestion :)

I move the function in umake/frameworks/__init__.py and I had annotated the function, but at the moment the description of the return is false. I began to implement the json dictionnary return, can you tell me if I am on the good way?